LayoutTests/fast/scrolling/smooth-scroll-with-overflow-hidden.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ AsyncOverflowScrollingEnabled=true ] -->
```

Comment:
This test may only work on macOS (monitorWheelEvents does nothing on iOS) so you should either move it into the mac directory, or add TestExpectations lines for it.

Reply:
Added to mac folder.